cli/deployment.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/controller/client"
+)
+
+func init() {
+	register("deploy", runDeployments, `
+usage: flynn deploy
+
+Manage app deployments
+
+Commands:
+    With no arguments, shows a list of deployments
+
+Examples:
+
+	$ flynn deploy
```

Comment:
Yep, that was my original thought. I think I changed it to fit the initial issue but deployment does make more sense to me too.